### PR TITLE
add kubescape

### PR DESCRIPTION
**Pull request by developer-guy**
Signed-off-by: Batuhan Apaydın <batuhan.apaydin@trendyol.com>

PTAL @dwertent @slashben

### CODE REVIEW

**Comment by developer-guy** [comment]
I'm just not sure that kubescape needs `libgit2` as a run-time dependency; folx, @dwertent @slashben would you mind helping me with that?

**Comment by slashben** [comment]
We need to parse it to parse Git objects. The problem we haven't found any Golang native alternative to libgit2

**Comment by developer-guy** [comment]
so, it depends to libgit2 package at runtime, right?

**Comment by matthyx** [comment]
> so, it depends to libgit2 package at runtime, right?

it's not a runtime dependency... the build is static so you only need the binary file

**Comment by developer-guy** [comment]
> > so, it depends to libgit2 package at runtime, right?
> 
> it's not a runtime dependency... the build is static so you only need the binary file

thanks, @matthyx, I've updated the package accordingly ☝️

**Comment by developer-guy** [comment]
> > so, it depends to libgit2 package at runtime, right?
> 
> it's not a runtime dependency... the build is static so you only need the binary file

thanks, @matthyx, I've updated the package accordingly ☝️

File: kubescape.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package:
+  name: kubescape
+  version: 2.0.182
```
Comment (dwertent) [inline kubescape.yaml:3]:
```suggestion
  version: 2.0.183
```

File: Makefile [hunk 2]
```diff
@@ -366,6 +366,7 @@ $(eval $(call build-package,py3-s3transfer,0.5.1-r0))
 $(eval $(call build-package,py3-urllib3,1.26.13-r0))
 $(eval $(call build-package,py3-yaml,6.0-r0))
 $(eval $(call build-package,aws-cli,1.27.38-r0))
+$(eval $(call build-package,kubescape,2.0.182-r0))
```
Comment (dwertent) [inline Makefile:369]:
```suggestion
$(eval $(call build-package,kubescape,2.0.183-r0))
```

Comment (developer-guy) [reply Makefile]: thanks @dwertent, I've upgraded it accordingly.

File: kubescape.yaml (hunk 1 above)
Comment (developer-guy) [reply kubescape.yaml]: thanks @dwertent, I've upgraded it accordingly.

**Review by dwertent — APPROVED** [review]

**Review by dlorenc — APPROVED** [review]

**Review by rawlingsj — APPROVED** [review]

**Pull request merged**